resources/js/features/game-list/components/DataTableRowActions/DataTableRowActions.tsx

```diff
@@ -0,0 +1,80 @@
+import type { Row } from '@tanstack/react-table';
+import { MdClose } from 'react-icons/md';
+
+import { BaseButton } from '@/common/components/+vendor/BaseButton';
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { useWantToPlayGamesList } from '@/common/hooks/useWantToPlayGamesList';
+import { cn } from '@/utils/cn';
+
+/**
+ * If the table row needs to have more than one action, it should go into a menu.
+ * @see https://ui.shadcn.com/examples/tasks
+ */
+
+interface DataTableRowActionsProps<TData> {
+  row: Row<TData>;
+}
+
+export function DataTableRowActions<TData>({ row }: DataTableRowActionsProps<TData>) {
+  const { auth } = usePageProps();
+
+  const { addToWantToPlayGamesList, removeFromWantToPlayGamesList } = useWantToPlayGamesList();
+
+  const rowData = row.original as Partial<App.Platform.Data.GameListEntry>;
+  const gameId = rowData?.game?.id ?? 0;
+  const gameTitle = rowData?.game?.title ?? '';
+  const isInBacklog = rowData?.isInBacklog ?? false;
+
+  const handleToggleFromBacklogClick = () => {
+    // This should never happen.
+    if (!gameId) {
+      throw new Error('No game ID.');
+    }
+
+    if (!auth?.user && typeof window !== 'undefined') {
+      window.location.href = route('login');
+
+      return;
+    }
```

Comment:
Important implementation detail: if users aren't authenticated, they'll still see the add button. Clicking it will redirect them to login.

I'm not sure if we have any means of setting a callback URL so they return to this page after logging in, but I'm not sure that's too important. New users will need to register, and existing users probably already know where they are.